barcelona.yml

```diff
@@ -26,10 +26,7 @@ environments:
     name: barcelona
     image_name: public.ecr.aws/degica/barcelona
     before_deploy: rake db:migrate
-    scheduled_tasks:
-      # 10AM JST every week day
-      - schedule: cron(0 1 ? * MON-FRI *)
-        command: bin/chaos
+    scheduled_tasks: []
```

Comment:
You are removing all scheduled task now?

Reply:
No, sorry let me remove that